smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/IndexGenerator.java

```diff
@@ -115,26 +115,29 @@ private static void writeClientExports(
             writer.write("export * from \"./commands/" + symbolProvider.toSymbol(operation).getName() + "\";");
             if (operation.hasTrait(PaginatedTrait.ID)) {
                 hasPaginatedOperation = true;
-                String modulePath = PaginationGenerator.getOutputFilelocation(operation);
-                writer.write("export * from \"./$L\";", modulePath.replace(".ts", ""));
+                String modulePath = PaginationGenerator.getOutputFilelocation(operation)
```

Comment:
Write a private function remove source folder

Reply:
Added in [`384ecbd` (#434)](https://github.com/awslabs/smithy-typescript/pull/434/commits/384ecbd295020c21f5490ea0a2aed1323e5c73ba)